azure-mgmt-scheduler/src/main/java/com/microsoft/azure/management/scheduler/Job.java

```diff
@@ -93,33 +98,217 @@
          */
         interface WithStartTime {
             /**
+             * A job which will start right away.
```

Comment:
to be consistent with starting the description with a verb: "Specifies that the job should be started right away."